.github/ISSUE_TEMPLATE/feature-suggestion.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature suggestion
```

Comment:
Should this say "IDE Feature suggestion", with a note about putting language feature suggestions in csharplang?